src/_includes/partials/givefeedback.njk

```diff
@@ -0,0 +1,26 @@
+<section data-pagefind-ignore class="py-100 mt-400 mb-600 {% if removeBorder %} {% else %}by-sm b-dark {% endif %}">
```

Comment:
Can we change `py-100 mt-400 mb-600` to `py-75 mt-300 mb-900` please to match the new spacing classes?

Reply:
```suggestion
<section data-pagefind-ignore class="py-75 mt-300 mb-900 {% if removeBorder %} {% else %}by-sm b-dark {% endif %}">
```

Reply:
Done!